packages/eslint-plugin/README.md

```diff
@@ -84,6 +84,22 @@ Install [`eslint-config-prettier`](https://github.com/prettier/eslint-config-pre
 
 **Note: Make sure you have `eslint-config-prettier@4.0.0` or newer.**
 
+## Usage with Airbnb
+
+Airbnb has two configs, a base one [`eslint-config-airbnb`](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) and one that includes rules for React [`eslint-config-airbnb-base`](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb-base).
```

Comment:
It's the other way around! The base variant doesn't have any react

```suggestion
Airbnb has two configs, a base one [`eslint-config-airbnb-base`](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb-base) and one that includes rules for React [`eslint-config-airbnb`](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb).
```

Reply:
Oops. Copy paste error there 😄